src/test/groovy/net/rubygrapefruit/platform/file/SymlinkFileEventFunctionsTest.groovy

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.rubygrapefruit.platform.file
+
+import net.rubygrapefruit.platform.NativeException
+import net.rubygrapefruit.platform.internal.Platform
+import spock.lang.Ignore
+import spock.lang.Requires
+
+import static java.nio.file.Files.createSymbolicLink
+import static net.rubygrapefruit.platform.file.FileWatcherCallback.Type.MODIFIED
+
+@Requires({ Platform.current().macOs || Platform.current().linux || Platform.current().windows })
```

Comment:
Does using `assumeTrue` in `setup` work instead of duplicating the class level `@Requires`?

Reply:
Unfortunately it doesn't. I tried that and a number of other things. I think this is good enough.